src/App.jsx

```diff
@@ -218,8 +218,9 @@ function App() {
               color="secondary"
               variant="contained"
               sx={buttonStyle}
+              className="button_post"
```

Comment:
class name button_post is not conventional. If we follow BEM convention, we can name it as something like post__newpost__button.

Reply:
ok let me do this